### PR TITLE
fix: ensure multinamespace role name not broken by truncation

### PR DESCRIPTION
**Pull request by carlmontanari**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind bugfix

**What does this pull request do? Which issues does it resolve?** (use `resolves #<issue_number>` if possible) 
N/A

**Please provide a short message that should be published in the vcluster release notes**
Fixed an issue where vclusters in multinamespace mode with long release names would not get the second cluster role created

**What else do we need to know?** 
basically just a better version of the previous "fix", this time hopefully for real :D

### CODE REVIEW

**Review by matskiv — APPROVED** [review]
Thanks @carlmontanari !
